lib/matplotlib/axes/_base.py

```diff
@@ -1261,43 +1261,51 @@ def get_aspect(self):
 
     def set_aspect(self, aspect, adjustable=None, anchor=None):
         """
-        *aspect*
-
-          ========   ================================================
-          value      description
-          ========   ================================================
-          'auto'     automatic; fill position rectangle with data
-          'equal'    same scaling from data to plot units for x and y
-           num       a circle will be stretched such that the height
-                     is num times the width. aspect=1 is the same as
-                     aspect='equal'.
-          ========   ================================================
-
-        *adjustable*
-
-          ============   =====================================
-          value          description
-          ============   =====================================
-          'box'          change physical size of axes
-          'datalim'      change xlim or ylim
-          'box-forced'   same as 'box', but axes can be shared
-          ============   =====================================
+        Set the aspect of the axis scaling, i.e. the ratio of y-unit to x-unit.
```

Comment:
axis->Axes ?

Reply:
I think it's really the ratio of y-axis-scaling to x-axis-scaling, not the aspect of the Axes bounding box.